apps/playground-sveltekit/.env.example

```
@@ -0,0 +1,3 @@
+VITE_GITHUB_CLIENT_ID=
+VITE_GITHUB_CLIENT_SECRET=
+VITE_NEXTAUTH_URL=
```

```suggestion
VITE_NEXTAUTH_URL=
VITE_NEXTAUTH_SECRET=
```

See: https://next-auth.js.org/configuration/options#nextauth_secret